docs/source/getting-started/code/quickstart-context.rst

```diff
@@ -7,7 +7,8 @@
 # /init
 
 # demo
->>> laplace_mechanism = dp.space_of(float) >> dp.m.then_laplace(scale=1.)
+>>> space = dp.space_of(float)
```

Comment:
For consistency with the other APIs, lets write this out:

```suggestion
>>> space = dp.atom_domain(T=float), dp.absolute_distance(T=float)
```

While `space_of` is cool when it works, it is a bit magical and isn't universally applicable. I don't think we should remove it from the public APIs, but it should be de-prioritized. The context API can be magical, lets keep examples in the framework API explicit.